st3/sublime_lib/_util/temporary_package.py

```diff
@@ -0,0 +1,30 @@
+import shutil
```

Comment:
A module or class docstring for this would be nice, since its use cases isn't immediately obvious compared to the other parts of this library, imo.